src/ceph-disk/ceph_disk/main.py

```diff
@@ -3789,6 +3789,10 @@ def main_activate_space(name, args):
     if not os.path.exists(args.dev):
         raise Error('%s does not exist' % args.dev)
 
+    if is_suppressed(args.dev):
```

Comment:
I am sorry. If its already in master on latest clone. this should be Mergerd/closed.